contracts/zero-ex/contracts/test/foundry/Local0xSwap.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2022 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+
+import "./utils/ForkUtils.sol";
+import "./utils/TestUtils.sol";
+import "./utils/DeployZeroEx.sol";
+import "forge-std/Test.sol";
+import "src/IZeroEx.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IEtherTokenV06.sol";
+import "src/features/TransformERC20Feature.sol";
+import "src/external/TransformerDeployer.sol";
+import "src/transformers/WethTransformer.sol";
+import "src/transformers/FillQuoteTransformer.sol";
+import "src/transformers/bridges/BridgeProtocols.sol";
+import "src/transformers/bridges/EthereumBridgeAdapter.sol";
+import "src/IZeroEx.sol";
+
+contract BasicSwapTest is Test, ForkUtils, TestUtils {
```

Comment:
This is just a WETH Transformer TransformERC20? I don't think the naming makes sense here (both `Local0xSwap` and `BasicSwapTest`). 

`test/foundry/transformERC20/WethTransformer.sol` ?